packages/salesforce-adapter/src/filters/settings_type.ts

```diff
@@ -46,6 +43,10 @@ const createSettingsType = async (
       childTypeNames: new Set(),
       client,
       isSettings: true,
+      annotations: {
+        suffix: SETTINGS_METADATA_TYPE.toLowerCase(),
+        dirName: SETTINGS_METADATA_TYPE.toLowerCase(),
```

Comment:
nit: these values are generally decoupled from their metadata type usually, so making them use the same constant is creating coupling where technically there should be none
```suggestion
        suffix: 'settings',
        dirName: 'settings',
```